### PR TITLE
Databases week2

### PR DESCRIPTION
**Pull request by hparul**

### CODE REVIEW

**Review by Elia-Git — COMMENTED** [review]
Pls, work on it thoroughly especially in PART 3 most of your queries are not reflecting the requested output. Reach out to anyone around you in the HYF community if you are in doubt.

I am very much looking to see your implementation to given feedback!

File: databases/week2/task2.sql [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+CREATE UNIQUE INDEX idx_sname on student(name);
+ALTER TABLE class
+ADD status VARCHAR(255) NOT NULL;
+UPDATE class
+set status="not-started" where id=1 OR id=4 or id=7 or id=9;
+UPDATE class
+set status="ongoing" where id=2 OR id=5 or id=8 ;
+UPDATE class
+set status="finished" where id=3 OR id=6 ;
+select * from class;
```
[the remark this answers is not in the document]
Comment (hparul) [reply databases/week2/task2.sql]: done

File: databases/week2/task1.sql [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+update
+  task
+set
+  task.status_id =(
+    select
+      status.id
+    from
+      status
+    where
+      status.name = 'Done'
```
[the remark this answers is not in the document]
Comment (hparul) [reply databases/week2/task1.sql]: done

File: databases/week2/task3.sql [hunk 3]
```diff
@@ -0,0 +1,34 @@
+use hyf_lesson2;
+--Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT title,task_id,user_id from task 
+join user_task on task.id=user_task.user_id;
```
[the remark this answers is not in the document]
Comment (hparul) [reply databases/week2/task3.sql]: resolved!

**Pull request merged**